addOns/quickstart/src/main/java/org/zaproxy/zap/extension/quickstart/launch/ExtensionQuickStartLaunch.java

```diff
@@ -233,6 +233,13 @@ protected void launchBrowser(String browserName, String url) {
                                     params.getConfig().save();
                                 }
                             } catch (Exception e1) {
+                                if (e1.getMessage().contains("cannot find")) {
+                                    View.getSingleton()
+                                            .showWarningDialog(
+                                                    Constant.messages.getString(
+                                                            "quickstart.explore.browser.not.found",
+                                                            browserName));
```

Comment:
It would be better to use `ExtensionSelenium#getWarnMessageFailedToStart`.

Reply:
![image](https://user-images.githubusercontent.com/7570458/145575519-f64cf9f4-94a6-4b60-acf2-afaef43d2a6c.png)
Doesn't seem terribly helpful?

Reply:
I'm suggesting it should use that method instead of adding the logic here. That method is used by the AJAX Spider, and should be by any tool that starts browsers this way (for consistent error handling and error messages).

Reply:
Oh okay, sorry missed what you were suggesting.

Reply:
I guess I'm still unclear where the split is. Are you suggesting the warning dialog be shown in that method? Or that the dialog still be shown here but the specific message/conditional handled in the Selenium add-on?

Reply:
I decided option 2 made the most sense. Current push fd26bef is done that way.